Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Bugzilla123456.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+
+#if UITEST
+using Xamarin.UITest;
+using NUnit.Framework;
+#endif
+
+namespace Xamarin.Forms.Controls.Issues
+{
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Bugzilla, 59925, "Issue Description", PlatformAffected.Default)]
```

Comment:
Not a blocker, but we might want that issue description fixed.

Reply:
Wtf I know I changed that...